src/SectionMeetings.tsx

```diff
@@ -0,0 +1,163 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import * as Moment from 'moment';
+import { IMeeting } from './types';
+
+interface Props {
+  meetings: Array<IMeeting>;
+}
+
+const styles = {
+  container: {
+    flex: 1,
+    wordWarp: 'normal',
+    justifyContent: 'flex-start',
+    div: {
+      display: 'inline-block',
+    },
+  },
+  boldedText: {
+    fontWeight: 700,
+  },
+};
+
+const SectionMeetings: React.FC<Props> = ({ meetings }) => {
+  // Correct for null meeting arrays
+  let correctedMeetings = meetings ?? [];
+  let sortedSectionMeetings = correctedMeetings.sort((a, b) =>
+    Moment.default(a.beginDate).isBefore(Moment.default(b.beginDate)) ? 1 : -1
+  );
+  console.log(sortedSectionMeetings[1]);
+  return (
+    <div css={styles.container}>
+      {sortedSectionMeetings.length === 1 && (
+        <div>
+          <div css={styles.boldedText}>
+            {Moment.default(sortedSectionMeetings[0].beginDate).format('dddd')}s
```

Comment:
Moment is a great choice for this, but I'd also take a look at the [Intl API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). It's a really cool new browser API that simplifies a lot of this.

Reply:
(For this I think Moment is the more mature tool but Intl is really neat)

Reply:
Oh, that is quite cool! Thanks, didn't know about it